AfterClass-io/afterclass.io-v2
# feat(ui): implement infinite scrolling

## Context



follow up to #144 and #194 

## Changes



- `fetchNextPage` triggered when a loading icon is brought into view
- refactor `<ReviewItemLoader/>` to make it less repetitive

## How to Test



1. go to `/`
2. scroll down
3. see icon and new review items should load soon after

## Preview / Screenshots




## Checklist
